firebase.json

```diff
@@ -160,37 +152,35 @@
+      { "source": "/install/**", "destination": "/install", "type": 301 },
       { "source": "/language-tour", "destination": "/guides/language/language-tour", "type": 301 },
       { "source": "/logos", "destination": "https://github.com/dart-lang/logos", "type": 301 },
+      { "source": "/linter/lints/:lint*", "destination": "http://dart-lang.github.io/linter/lints/:lints*", "type": 301 },
       { "source": "/mailing-list", "destination": "https://groups.google.com/a/dartlang.org/forum/#!forum/misc", "type": 301 },
       { "source": "/mobile", "destination": "/guides/platforms", "type": 301 },
+      { "source": "/news", "destination": "/", "type": 301 },
+      { "source": "/news/**", "destination": "/", "type": 301 },
+      { "source": "/observatory/**", "destination": "https://dart-lang.github.io/observatory", "type": 301 },
       { "source": "/performance", "destination": "/articles/dart-vm/benchmarking", "type": 301 },
       { "source": "/performance/io", "destination": "/articles/dart-vm/benchmarking", "type": 301 },
-      { "source": "/polymer", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
-      { "source": "/polymer-dart/**", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
-      { "source": "/polymer-old", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
-      { "source": "/polymer-old/**", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
-      { "source": "/polymer/reference/error-messages", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
-      { "source": "/polymer/spa", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
-      { "source": "/polymer/using-elements", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
+      { "source": "/polymer*", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
+      { "source": "/polymer*/**", "destination": "https://webdev.dartlang.org/guides/polymer", "type": 301 },
       { "source": "/posters/**", "destination": "https://github.com/dart-lang/www.dartlang.org/tree/f5f235e56e51d38d9f4d01f949d0a485f3828056/src/site/posters", "type": 301 },
       { "source": "/pub/tools/editor", "destination": "/tools", "type": 301 },
       { "source": "/redirects/sdk-download-dev", "destination": "/install", "type": 301 },
       { "source": "/redirects/sdk-download-stable", "destination": "/install", "type": 301 },
-      { "source": "/resources/dart-tips", "destination": "/guides/language/language-tour", "type": 301 },
-      { "source": "/resources/dart-tips-ep-2/**", "destination": "/guides/language/language-tour", "type": 301 },
-      { "source": "/resources/dart-tips/**", "destination": "/guides/language/language-tour", "type": 301 },
+      { "source": "/resources/dart-tips*", "destination": "/guides/language/language-tour", "type": 301 },
```

Comment:
I had combed the list to ensure that this covered all cases. I'll add a trailing `{,/**}` just in case. Anything else you believe needs to be covered?